bin/build-cored-release

```diff
@@ -0,0 +1,57 @@
+#!/bin/sh
+set -e
```

Comment:
If we use `set -eu` the `u` will fail the script if any variables are unset. I think its a good case to always protect against.